<launch>
    <node pkg="ws281x" name="led" type="ws281x_node" clear_params="true" output="screen">
        <param name="led_count" value="60"/>
        <param name="gpio_pin" value="13"/>
        <param name="brightness" value="255"/>
        <param name="strip_type" value="WS2811_STRIP_GRB"/>
        <param name="target_frequency" value="800000"/>
        <param name="dma" value="10"/>
        <param name="invert" value="false"/>
    </node>
</launch>
